content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Reporting Screens"
+description: "This article provides information on TrueNAS reporting graph screens and settings."
+weight: 10
+alias: /scale/scaleuireference/reporting/
+tags:
+ - scalereports
+---
+
+
+{{< toc >}}
+
+
+The **Reporting** screen displays graphs of system information for CPU, disk, memory, network, NFS, partition, target, UPS, ZFS and system functions. The **CPU** report displays by default.
+
+The <i class="material-icons" aria-hidden="true" title="Settings">settings</i> opens the **[Reports Configuration](#reports-configuration-screen)** configuration screen.
+
+## Reports Configuration Screen
+**Reports Configuration** settings specify how TrueNAS displays the graphs and the host name of the Graphite server.
```

Comment:
```suggestion
**Reports Configuration** settings specify how TrueNAS displays the graphs and the hostname of the Graphite server.
```

Reply:
host name in this instance is two words because it is not the CLI command